geopandas/tests/test_sindex.py

```diff
@@ -22,8 +22,20 @@ class TestNoSindex:
     @pytest.mark.skipif(sindex.has_sindex(), reason="Spatial index present, skipping")
     def test_no_sindex_installed(self):
         """Checks that an error is raised when no spatial index is present."""
+        t1 = Polygon([(0, 0), (1, 0), (1, 1)])
```

Comment:
Suggest adding a bit more to the docstring to clarify what is being tested here:
```
Check that an `ImportError` is raised when attempting to generate a spatial index for a GoeDataFrame when the spatial index implementation library is not available.  Confirm that the index is not generated by the failed attempt to generate it.
```